pytorch/training/docker/1.6.0/py3/cu101/Dockerfile.gpu

```diff
@@ -172,6 +205,18 @@ RUN apt-get update \
  && mv /etc/ssh/ssh_config.new /etc/ssh/ssh_config \
  && rm -rf /var/lib/apt/lists/*
 
+# Install OFI Plugin
+RUN cd /tmp && \
+  git clone https://github.com/aws/aws-ofi-nccl.git -b v1.0.1-aws && \
+  cd aws-ofi-nccl && \
+  ./autogen.sh && \
+  ./configure --with-libfabric=/opt/amazon/efa --with-mpi=${OPEN_MPI_PATH} \
+  --with-cuda=${CUDA_HOME} \
+   --with-nccl=/usr/local --prefix=/usr/local && \
+  make && \
+  make install && \
+  rm -rf /tmp/aws-ofi-nccl
```

Comment:
Please re-format this `RUN` statement to match the other `RUN` statements, such as the one in lines 188-196.

Reply:
Reformated.